.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches:
+    - main
+    - master
+    tags:
+    - 'v*'
+
+name: Release policy
+
+jobs:
+
+  test:
+    name: run tests and linters
+    uses: kubewarden/github-actions/.github/workflows/reusable-test-policy-rust.yml@v4.0.0
```

Comment:
We should be consuming >= 4.5.0.

(opened https://github.com/kubewarden/rust-policy-template/issues/54 for the template itself).